css/main.css

```diff
@@ -11,6 +11,15 @@
            padding-bottom: 20px;
        }
 
+    /*Home page styling*/
+    .row {
+      font-size: 24px;

```

Comment:
Off to a good start!  Already makes the site jump out a bit more, though I do think it would be better to differentiate the headers a little bit more since they now look very close to how the regular text does.